basics/plugin_structure/plugin_components.md

```diff
@@ -29,25 +29,25 @@ Application level components must be registered in the `<application-components>
 
 ## Project level components
 
-Optionally, a project level component's implementation class may implement the [ProjectComponent](upsource:///platform/core-api/src/com/intellij/openapi/components/ProjectComponent.java) interface.
+Optionally, a project level component's implementation class may implement the [ProjectComponent](upsource:///platform/projectModel-api/src/com/intellij/openapi/components/PersistentStateComponent.java) interface.
```

Comment:
Should this be pointing at `PersistentStateComponent.java`, or should the name of the link change to `PersistentStateComponent`?

Reply:
This should *not* be pointing at `PersistentStateComponent`; that's a completely different interface.

Reply:
Righto. I'll revert this change, and merge the rest.